### DISCUSSION TITLE
GameCube internal wiring

### DISCUSSION THREAD

**Discussion by Zonark** [darthcloud/BlueRetro#436] [Q&A]
Just have a small question with the GameCube internal install. I just need the data lines on each port in order to install this correctly correct?
I am using the suggested esp32 board as suggested by the creator and used the links to purchase them.
here’s the catch
I have a trimmed down GameCube with a trimmed down controller port PCB. (Only traces relocated are the battery, led, and reset switch. 

this photo I think is the correct wiring from what I can tell and all of the traces for all ports are intact.

![7D8FB431-4DF6-4651-BB9E-4C47CCF689C3](https://user-images.githubusercontent.com/42153163/196981089-d8f4644c-e39f-4d63-9dc6-af3b0b9f1d3f.jpeg)

I see some people with buttons added to the outside of their cases. My question is also what are these for?

and do I need any resistors or anything

**Comment by OldManGamingRetro** [comment]
whats the best picture to show where wires install to?  controller board and esp chip.
